app/assets/stylesheets/pin-index.css.scss

```diff
@@ -0,0 +1,130 @@
+$primary-background-color: rgba(237, 242, 242, 0.8);
+
+
+#container{
+  background-color: $primary-background-color;
+}
+
+hr{
+  height: 0;
+  max-height: 0;
+  font-size: 1px;
+  line-height: 0;
+  clear: both;
+  color: $primary-background-color;
+  border: none;
+  border-top: 1px solid #aaaaaa;
+  border-bottom: 1px solid #ffffff;
+}
+
+.pin-outer{
+  background-color: white;
+  border: solid $primary-background-color 2px;
+  max-width: 220px;
+  min-height: 300px;
+  background-color: $primary-background-color; 
+}
+
+div.pin{
+  margin: 0px;
+  border-radius: 10px;
+}
+
+div.thumbnail{
+  border-radius: 10px;
+  padding: 0px;
+}
+
+div.picture-wrap {
+  
+  &:hover{
+    cursor: wait;
+  }
+}
+
+.picture-wrap:hover .panel-visible{
+  display: inherit;
+}
+
+.panel-visible{
+  height: 100%;
+  position: relative;
+  display: none;
+}
+
+.like-button{
+  text-align: right;
+  padding: 10px;
+  &:hover{
+  }
+}
+
+.url{
+  text-align: left;
+  font-size: 11px;
+  padding: 5px;
+  position: absolute;
+  bottom: 5px;
+  a:hover{
+    text-decoration: none;
+    color: white;
+  }
+}
+
+a.link{
+  color: $primary-background-color;
+  padding-left: 5px;
+}
+
+button.heart{
+  &:hover{
+  // cursor: wait;
+  }
+}
+
+.image{
+  width: 100%;
+  height: 100vh;
+  max-height: 500px;
+  align-content: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  &:hover{
+  // cursor: wait;
+  }
+}
+
+div.thumbnail .caption{
+  padding: 0px;
+}
+
+h2.category{
+  padding-left: 20px;
+  margin-top: 20px;
+  margin-bottom: 10px;
+  font-size: 10px;
+  overflow-wrap: break-word;
+  color: $primary-background-color;
+}
+
+h1.title{
+  border-bottom: 1px $primary-background-color solid;
+  font-size: 14px;
+  padding: 10px;
+  padding-left: 20px;
+  padding-bottom: 20px;
+  margin-top: 10px;
+  overflow-wrap: break-word;
+}
+
+.description{
+  font-size: 11px;
+  padding: 20px;
+  &:hover {
+    // cursor: wait;
+  }
+}
+
```

Comment:
same thing on the pin-index css file. should not be onthis branch. probably because i used pin-index branch as base when i checkout to pin-show branch....